### PR TITLE
Update Verticle documentation

### PR DESCRIPTION
**Pull request by fussel178**
### Summary 

Update tutorials and concepts to use the new `TelestionVerticle` class.

### Details 

A little longer ago we introduced our own verticle class named `TelestionVerticle` and traits to enrich application verticles.
This PR tries to update the existing documentation about writing and deploying verticles.

### Additional information 

n/a

### CLA

- [x] I have signed the individual contributor's license agreement **and sent** it to the board of the WüSpace e. V. organization.

### CODE REVIEW

**Comment by fussel178** [comment]
@pklaschka You can take a first review round. Please don't merge this PR because I'm working on a tutorial how to use traits for verticles. :wink:

**Review by pklaschka — COMMENTED** [review]
@fussel178 Awesome! I think if you introduce traits, it makes sense to add a "Trait" concept, as well, which quickly explains what traits are, how one can generally use them, how to identify them ("starts with `With`"), and maybe even on how they are implemented ("you can even implement your own traits by ...").

File: docs/application/concepts/130-traits.mdx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+--------
+
+<Reference to="../other-article">
+		Relative Link to other article
+</Reference>
+
+<Reference to="https://www.example.com">
+		Example Website
+</Reference>
+-->
```
Comment (pklaschka) [inline docs/application/concepts/130-traits.mdx:123]: <i class='graphite__hidden'>[Re: lines 110 to 124]<br/><br/></i>

Let's add a link to the API Reference of all available `TelestionVerticle` traits, here, please: https://javadoc.io/doc/de.wuespace.telestion/telestion-api/latest/de/wuespace/telestion/api/verticle/trait/package-summary.html

Comment (fussel178) [reply docs/application/concepts/130-traits.mdx]: Done!

**Review by pklaschka — APPROVED** [review]
LGTM. Thank you for your contribution 🎉 🙂

**Pull request merged**